src/server/api/user.ts

```diff
@@ -105,6 +106,23 @@ function validateState(req: Express.Request, res: Express.Response, next: Expres
   next()
 }
 
+/**
+ * Make sure all parameters needed for the pre-signed url request are present.
+ */
+function validatePresignedUrlRequest(
+  req: Express.Request,
+  res: Express.Response,
+  next: Express.NextFunction,
+) {
+  if (!req.body.fileType || !req.body.key) {
+    res.badRequest(
+      jsonMessage('Some or all required arguments are missing: fileType, key.')
+    )
+    return
```

Comment:
nitpicking here, but it's good practice write it out like this
```suggestion
    res.badRequest(
      return jsonMessage('Some or all required arguments are missing: fileType, key.')
    )
```